dir/path.go

```diff
@@ -36,13 +36,28 @@ const (
 	TrustStoreDir = "truststore"
 )
 
+// WriteLevel defines the write directory level supporting UserLevel or SystemLevel.
+type WriteLevel int
+
+const (
+	// SystemLevel is the label to specify write directory to system level
+	SystemLevel WriteLevel = 0
+	// UserLevel is the label to specify write directory to user level
+	UserLevel WriteLevel = 1
+)
+
```

Comment:
Why levels are specific to write? can we make them generic so they be used for read too?

Reply:
Renamed `WriteLevel` to `DirLevel`